specs/core/0_beacon-chain.md

```diff
@@ -1389,11 +1388,12 @@ Below are the processing steps that happen at every slot.
 
 * Set `state.slot += 1`.
 * Set `state.validator_registry[get_beacon_proposer_index(state, state.slot)].randao_layers += 1`.
+* Set `state.latest_randao_mixes = state.latest_randao_mixes[(block.slot - 1) % LATEST_RANDAO_MIXES_LENGTH]`
```

Comment:
* This needs to be `state.slot`. You are not guaranteed to have a `block` here.
* missing index into `latest_randao_mixes` on the left side of the assignment

```suggestion
* Set `state.latest_randao_mixes[state.slot % LATEST_RANDAO_MIXES_LENTH] = state.latest_randao_mixes[(state.slot - 1) % LATEST_RANDAO_MIXES_LENGTH]`
```